scripts/mods/mod_reforged/dynamic_spawns/parties/barbarian_parties.nut

```diff
@@ -21,9 +45,31 @@ local parties = [
 		VisibilityMult = 1.0,
 		VisionMult = 1.0,
 		UnitBlockDefs = [
-			{ BaseID = "UnitBlock.RF.BarbarianHunterFrontline", RatioMin = 0.60, RatioMax = 1.0, DeterminesFigure = true },
+			{ BaseID = "UnitBlock.RF.BarbarianFrontline", RatioMin = 0.60, RatioMax = 1.0, DeterminesFigure = true },
```

Comment:
Is it intended that you break with vanilla spawn behavior here?
`BarbarianHunters` only spawn thralls and dogs in vanilla

Reply:
Yes, because our parties scale longer than vanilla I thought this would be preferable to massive thrall parties.

Reply:
How about we keep BarbarianHunterFrontline block but add new units to that?
That way the uniqueness of a BarbarianHunters party can still be preserved. It's weird to see fully armored Chosen moving around as "Hunters"